python/ray/tests/test_reference_counting_2.py

```diff
@@ -289,7 +289,7 @@ def recursive(num_tasks_left):
     _fill_object_store_and_get(final_oid_bytes, succeed=False)
 
 
-@pytest.mark.parametrize("failure", [False, True])
+@pytest.mark.parametrize("failure", [True, False])
```

Comment:
any reason for the swapping?

Reply:
Oh there's no reason... I tested something, and it was reverted. Will fix